site/viamguides/src/ai-camera-tracking/ai-camera-tracking.md

```diff
@@ -0,0 +1,423 @@
+
+### Configure your base
+
+1. In [the Viam app](https://app.viam.com/fleet/locations) under the **CONFIGURE** tab, click the **+** icon in the left-hand menu and select **Component**.
+1. Select `base`, and find the `pan-tilt-camera:base-control` module. This adds the module for sending directions to a pair of pan-tilt servos from a single component. Leave the default name `base-1` for now.
+   ![add base component](assets/addBase.webp)
+1. In the new `base-1` panel, configure your component with the following JSON in the **CONFIGURE** field. This tells your base component which servos to use for the pan and tilt functionalities.
+   ```json
+   {
+       "pan": "pan",
+       "tilt": "tilt"
+   }
+   ```
+   ![configure base component](assets/configureBase.webp)
+1. Click **Save** in the top right. This may take a moment to apply your configuration changes.
+1. At the bottom of the `base-1` panel, expand the **TEST** section to ensure you have configured the base properly. After toggling the keyboard control under the **Quick move** section, you should see the appropriate servo move when using your keyboar's arrow keys or WASD.
+   ![test base](assets/testBaseQuickMove.webp)
+
+> aside negative
+> **TROUBLESHOOTING**: Double check the wiring on your components. Also check under the **LOGS** tab to see what might be going wrong.
+
+With your hardware components confirmed working, now we will provide the intelligence to enable real-time object tracking!
+
+<!-- ------------------------ -->
+## Configure your services
+Duration: 5
+
+### Add an ML model service
+
+1. In the Viam app, click the **+** icon in the left-hand menu and select **Service**, and then `ML model`.
+1. Search for a module called `TFLite CPU`. Then click **Add module**, and **Create** a new [ML Model service](https://docs.viam.com/services/ml/) called `mlmodel-1`. This provides the ability to run Tensorflow Lite, a.k.a [LiteRT](https://ai.google.dev/edge/litert), models on the machine.
+   ![add ML model](assets/addMLModel.webp)
+1. Notice this creates two new items in the left sidebar. The first is your new ML Model service called `mlmodel-1`, and the second is the `tflite_cpu` module from the Registry.
+1. In the `mlmodel-1` panel, click "Select model" in the **Model** section, search for "EfficientDet-COCO" under the **Registry** tab, and select the model from the list.
+   ![configure model](assets/configureMLModel.webp)
+   ![search model](assets/searchMLModel.webp)
+   ![select model](assets/selectMLModel.webp)
+1. Click **Save** in the top right to save and apply your configuration changes.
+
+### Add a vision service
+
+1. In the Viam app, click the **+** icon in the left-hand menu and select **Service**, and then `vision`.
+1. Search for a module called `mlmodel`. Then click **Add module**, and **Create** a new [Vision service](https://docs.viam.com/services/vision/) called `vision-1`.
+   ![add vision module](assets/addVisionService.webp)
```

Comment:
Consider changing the `addVisionService.webp` screenshot to show the search term as `mlmodel` instead of `vision`. In this way, the screenshot matches the step it follows, preventing confusion.